src/goSuggest.ts

```diff
@@ -50,8 +50,22 @@ export class GoCompletionItemProvider implements vscode.CompletionItemProvider {
 				let lineTillCurrentPosition = lineText.substr(0, position.character);
 				let autocompleteUnimportedPackages = config['autocompleteUnimportedPackages'] === true && !lineText.match(/^(\s)*(import|package)(\s)+/);
 
-				if (lineText.match(/^\s*\/\//)) {
-					return resolve([]);
+				if (lineText.match(/\//)) {  // complete exported item`s comment
```

Comment:
This should be `lineText.match(/^\/\//)` and regex test in the next line can be skipped

Other wise in the below scenario where `|` denotes the cursor position, no completions will be returned
`fmt.Print(|)  // Some comment`

Reply:
using `lineText.match(/^\/\//)` at first, it will sugguest alot of completions. My solution can skip one slash`s completion.